mne/io/kit/coreg.py

```diff
@@ -114,7 +114,7 @@ def read_sns(fname):
     return locs
 
 
-def _set_dig_kit(mrk, elp, hsp, eeg):
+def _set_dig_kit(mrk, elp, hsp, bad_coils, eeg):
```

Comment:
To make sure we don't mess anything up anywhere (including in [mne-kit-gui](https://github.com/mne-tools/mne-kit-gui)), we use `*,` to enforce that `bad_coils` must be passed as a keyword argument, not by position:
```suggestion
def _set_dig_kit(mrk, elp, hsp, eeg, *, bad_coils=()):
```
Also let's put the new kwarg at the end, and make the default backward compatible